### ISSUE TITLE
Docs build broken on readthedocs

### ISSUE THREAD

**Issue by bwohlberg** [lanl/scico#168]
The error occurs suspiciously close to
```
WARNING: Cannot resolve forward reference in type annotations of "scico.flax.ConvBNBlock": name 'Module' is not defined
```
(see #167) but this warning was not leading to build failure until today. It appears that this change is related to the recent release of `sphinx-autodoc-typehints` v1.15.

**Comment by bwohlberg** [comment]
Temporarily resolved by #169. Long term solution still needed.

It appears that this issue is due to recent changes in `sphinx-autodoc-typehints` making it less robust to the forward reference warning described in #167.

**Comment by Michael-T-McCann** [comment]
I tried to reproduce this both locally (with the fresh conda env as per installation instructions) and via a github action in #171 and could not. Thoughts on what I'm missing?
